std/array.d

```diff
@@ -3891,3 +3891,231 @@ unittest
     assert(appS.data == "hellow");
     assert(appA.data == "hellow");
 }
+
+version(all)
+{
+
+import std.traits : CommonType, Unqual, isStaticArray;
+import std.conv : emplaceRef;
+
+// @nogc: https://issues.dlang.org/show_bug.cgi?id=18439
+//nothrow @safe pure:
+nothrow:
```

Comment:
Please avoid using attributes like this, they are a huge pain later on and Andrei has expressed desire for removing all such uses from Phobos.

Reply:
done